Makefile

```diff
@@ -140,12 +140,12 @@ test-legacy: ## Run the legacy tests (go/v2) of the testdata directory
 	./test/testdata/test_legacy.sh
 
 .PHONY: test-e2e-local
-test-e2e-local: ## Run the end-to-end tests locally
+test-e2e-local: build-external-plugin ## Run the end-to-end tests locally
 	## To keep the same kind cluster between test runs, use `SKIP_KIND_CLEANUP=1 make test-e2e-local`
 	./test/e2e/local.sh
 
 .PHONY: test-e2e-ci
-test-e2e-ci: ## Run the end-to-end tests (used in the CI)`
+test-e2e-ci: build-external-plugin ## Run the end-to-end tests (used in the CI)`
```

Comment:
Revert this change you added it in the prow already

Reply:
I think this is still needed. IIUC, just the execution of the go tests were added in the Prow configuration

Reply:
The prow setup is done in this one: https://github.com/kubernetes-sigs/kubebuilder/blob/master/test/e2e/setup.sh#L56-L67

We do not call any of our tests in this way, so why we would need to have this new target and call in the test-e2e-ci?